metashare/api/models.py

```diff
@@ -90,3 +94,24 @@ def valid_token_for(self):
         if all(self.token) and self.org_id:
             return self.org_id
         return None
+
+
+class Product(HashIdMixin):
+    name = models.CharField(max_length=50, unique=True)
+    repo_name = models.SlugField(unique=True)
+    version_number = models.CharField(max_length=50)
+    description = MarkdownField()
+    is_managed = models.BooleanField(default=False)
+    license = ArrayField(
+        models.CharField(max_length=64, choices=LICENSES), blank=True, default=list
+    )
```

Comment:
Not sure this needs to be a field on the model; it's only going to be used as input when generating a new product's repository, right?

Also making it a list seems weird (_especially_ with a singular field name). Including multiple licenses in a repository is a bit of an edge case and it's fine if we have to tell people to edit files manually for that case.